src/models/transactions/common.ts

```diff
@@ -66,6 +66,10 @@ export function isAmount(amount: Amount): boolean {
   return typeof amount === "string" || isIssuedCurrency(amount);
 }
 
+export enum GlobalFlagsEnum {
+  tfFullyCanonicalSig = 0x80000000,
+}
+
```

Comment:
This flag isn't actually used anymore: https://xrpl.org/transaction-common-fields.html#global-flags

Reply:
Since this is deprecated and the only flag in `GlobalFlags`, we should remove `GlobalFlags` since it'll become an empty interface. @natenichols defined the `GlobalFlags` interface in another [commit](https://github.com/ripple/ripple-lib/commit/3231c9e08c6ff469876d5ff99093e32a72f12852). How do you feel about my suggestion?

Reply:
Since this is deprecated and the only flag in GlobalFlags, we should remove GlobalFlags since it'll become an empty interface. @natenichols defined the GlobalFlags interface in another commit. What do you think about my suggestion?

Reply:
We need to have a `GlobalFlags`, because otherwise we'd have to specify flags in the baseTransaction as `Flags: number | Record<string,unkown>` We can do that, but i think its a lot cleaner to have `GlobalFlags` serve as a base class.

Reply:
I'm fine with leaving it as an empty interface - it makes it easier to add other global flags if more get added in the future.